### DISCUSSION TITLE
Colt: CLI to run Bolt tasks through Choria

### DISCUSSION THREAD

**Discussion by neomilium** [choria-io/general#28] [Show and tell]
Hi,

Few months ago, I was using _Choria_ through `mco` CLI to run _Bolt_ tasks and found some _extra_ features lacks to me.

Features I was looking were:
 - Output continuously results/errors during task run
 - Enhance results display to help me to locate relevant information
 - Wrap some _MCO_ methods to task run easily when using it through another ruby code

So, I started to wrote [Colt](https://github.com/opus-codium/choria-colt) and, even its still in development, its now functional and daily used.

Have fun and feel free to contribute!

**Comment by neomilium** [comment]
Some features I may implement are written here: https://github.com/opus-codium/choria-colt/projects/2

**Comment by ripienaar** [comment]
Nice, I will share on the slack channel also
